serf/config.go

```diff
@@ -222,6 +222,10 @@ type Config struct {
 	// persist changes to the encryption keyring.
 	KeyringFile string
 
+	// DisableKeyringFile prevents serf from writing keyring data to the
```

Comment:
Could we use an empty `KeyringFile` name to indicate that this is disabled w/o a new config param?